trailblazer/clients/tower/models.py

```diff
@@ -94,6 +94,10 @@ def is_complete(cls) -> bool:
         return cls.status == SlurmJobStatus.COMPLETED
 
 
+class TaskWrapper(BaseModel):
+    task: TowerTask
```

Comment:
Why is a wrapper needed?

Reply:
The response data from tower is structured that way: https://cloud.seqera.io/openapi/#get-/workflow/-workflowId-/tasks.

We could hide it away by doing some pydantic magic, but I rather keep the structure close to the actual response structure to limit confusion. I did add a `get_tasks` utils method to the `TowerTasksResponse` model to discard the wrapper and just get the tasks.

Reply:
oof, if the other option is `tasks: list[dict[str, TowerTask]]` then i prefer this 😅